src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const cells = rows[i].cells;
+  const positionCells = cells[1];
+  const cloneCells = positionCells.cloneNode(true);
```

Comment:
Cloning the node is correct, but ensure that the cloned node is being inserted correctly and that the table structure supports this operation.